docs/design.md

```diff
@@ -104,6 +104,17 @@ These blocks are called _address blocks_, and assigned to nodes.  Since all IP a
 
 For example, if an address pool defines that the size of an address block is 2<sup>5</sup>, `coil-controller` will curve an address block for IPv4 with `/27` subnet mask out of the pool, and assigns it to a node.
 
+
+When assigning an address, if `coil-controller` reuses the last released address immediately, it tends to use old routes.
```

Comment:
> When assigning an address, if `coil-controller` reuses the last released address immediately, it tends to use old routes.

I don't get what this sentence means. Why "it tends to use old routes"??
I think this paragraph can make sense without this sentence.

Reply:
I wanted to show some examples of concerns of reusing addresses.
But I certainly may not need it.